adoc/chapters/programming_interface.adoc

```diff
@@ -18860,6 +18856,46 @@ corresponds to the position [code]#index# of the swizzle operation.
 
 '''
 
+[frame=all,grid=none,separator="@"]
+|====
+a@
+[source]
+----
+const __writeable_swizzle__&
+operator=(const __writeable_swizzle__& rhs) const
```

Comment:
Why is this a `const` member function?

Reply:
It is still logically a non-`const` operation.

There is precedent in C++ for this, such as `ostream_iterator::operator=(const T&)` (as well as some of its other operators).